packages/gatsby-transformer-react-docgen/package.json

```diff
@@ -10,15 +10,15 @@
     "@babel/code-frame": "^7.5.5",
     "@babel/runtime": "^7.5.5",
     "@babel/types": "^7.5.5",
-    "ast-types": "^0.12.4",
+    "ast-types": "^0.13.2",
```

Comment:
this is wrong, actually cc @jquense 

this was a breaking change. this package needs ^0.12

Reply:
Do you mind pointing out what the breaking changes were? There doesn't seem to be a changelog on that package which makes it hard to find what was a breaking change.

Reply:
The issue is that react-docgen itself is still on 0.12, and there's an implicit requirement that this plugin be on the same version of ast-types as react-docgen.

Reply:
yeah, `react-docgen` should make this a peer dependency ideally. If you end up with 0.13 and 0.12 in a project it'll break.

Reply:
Ah, thanks @jquense 

Fixed in https://github.com/gatsbyjs/gatsby/commit/ff7610f542e5ad96e86917a7894e18aea3579d32